tests/test_ColorsDataset.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from torch import Tensor
+from color_generator.datasets import ColorsDataset
+
+
+@pytest.fixture
+def dataset():
+    return ColorsDataset()
```

Comment:
ColorsDataset depends entirely on [_load_and_process_colors()](https://github.com/wronaq/color-generator/blob/84f9bcc308143fd12d93b65af5f1ffb1e7f2ead2/color_generator/datasets/colors_dataset.py#L43) function and this test would fail on any other dataset - we should parameterize `path_to_dataset` and pass the current path here (we could even copy that file to test directory).